extensions/amp-form/0.1/form-validators.js

```diff
@@ -351,6 +361,8 @@ export function getFormValidator(form) {
       return new AsYouGoValidator(form);
     case CustomValidationTypes.ShowAllOnSubmit:
       return new ShowAllOnSubmitValidator(form);
+    case CustomValidationTypes.AsYouGoShowAllOnSubmit:
```

Comment:
How about we make `custom-validation-reporting` take multiple space-separated values, create a unified validator that takes the type of the validation(s) as a parameter and get rid of `InteractAndSubmitValidator`. I think the infrastructure is already there for this to happen and that way the syntax would be `custom-validation-reporting = 'as-you-go show-all-on-submit'`.